portal-web/docroot/html/taglib/ui/input_localized/page.jsp

```diff
@@ -155,7 +155,7 @@
 
 							Locale curLocale = LocaleUtil.fromLanguageId(curLanguageId);
 
-							if (errorLocales.contains(curLocale) || ((index == 0) && errorLocales.isEmpty())) {
+							if (curLanguageId.equals(selectedLanguageId)) {
```

Comment:
The changes looks fine but I'm a bit afraid to go removing `errorLocales` check here because I'm not sure why it was implemented.

Looks like the implementation before https://github.com/liferay-frontend/liferay-portal/commit/a07cea1a05edd18578e6790c72bb7380e19c31c2 was a bit similar to it.

See: 
https://github.com/liferay-frontend/liferay-portal/commit/a07cea1a05edd18578e6790c72bb7380e19c31c2#diff-ddf7ea79e2bd87824260b83c1ead456b54a9166d16ac19bd4188e01dab062c19L154-R175

I would ping @jbalsas to see which side-effects of removing `errorLocales` can be raised here.

Reply:
Look like `errorLocales` was created to handling exceptions but I don't know how to reproduce the use case to have an error :/

See: 
https://github.com/liferay-frontend/liferay-portal/commit/1563310e1899f08da3757439d375d22ec7501c52

_LPS-42184 Localizer taglib will transparently get the errorLocales if there is any Exception whose focusField matches the fieldName. In that case, if the exception extends LocalizedException it will get the errorLocales in order to show the errors._

Reply:
My fix could be modified to not remove the errorLocales but only the ((index == 0) part and replace it with my fix.
What do you think about it?

Reply:
So... this code might be older than myself but if I guess correctly, we want to focus on the correct input in case there is a validation error in one of them.

For example, if we're setting friendly urls and one of them does not have a `/` we want it focused and active to simplify correcting the mistake.

I think this might not even be working as we speak, but I think that's the design we have here.

I'd say this can be adjusted to:
- If `errorLocales.contains(curLocale)` -> set it (might be edge cases here...)
- If `errorLocales.isEmpty` set it when it's the same as `selectedLanguageId` which is guaranteed to always have a value defaulting to `defaultLanguageId`

Reply:
Thank you I will modify my code according to this.

Reply:
Thanks @bakayattila, feel free to push to this branch and let us know when ready to go over it again!